README.md

```diff
@@ -67,143 +64,68 @@
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
+- **A responsive webpage to present my knowledge, experience and projects in programming.**
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
+- [Live Demo Link](https://github.com/CarlosZ96/Portfolio)
```

Comment:
- [ ] [OPTIONAL] Kindly remove the live demo link since the project has not been deployed yet. 🛩️